### PR TITLE
ACR API version 2021-09-01

### PR DESCRIPTION
**Pull request by AntonioVT**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Add a changelog entry for this PR by answering the following questions:
  1. What's the purpose of the update?
      - [ ] new service onboarding
      - [x] new API version
      - [ ] update existing version for new feature
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify
  2. When are you targeting to deploy the new service/feature to public regions? Please provide the date or, if the date is not yet available, the month.
  3. When do you expect to publish the swagger? Please provide date or, the the date is not yet available, the month.
  4. If updating an existing version, please select the specific langauge SDKs and CLIs that must be refreshed after the swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No refresh required for updates in this PR

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist 

> **Applicability**: :warning: 
>
> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
> - Change to data plane APIs 
> - Adding new properties 
> - All removals

Otherwise your PR may be subject to ARM review requirements. Complete the following:
- [ ] Check this box if any of the following apply to the PR so that label “WaitForARMFeedback” will be added automatically to begin ARM API Review. Failure to comply may result in delays to the manifest.
  - Adding a new service
  - Adding new API(s)
  - Adding a new API version
    -[ ] To review changes efficiently, ensure you copy the existing version into the new directory structure for first commit and then push new changes, including version updates, in separate commits.

- [ ] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If any of the following scenarios apply to the PR, request approval from the Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 
- [ ] Removing API(s) in a stable version
- [ ] Removing properties in a stable version
- [ ] Removing API version(s) in a stable version
- [ ] Updating API in a stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2021-09-09/examples/ImportImageByTag.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "parameters": {
+    "api-version": "2021-06-01-preview",
```
Comment (jaysterp) [inline specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2021-09-09/examples/ImportImageByTag.json:3]: @AntonioVT please update these examples to use `2021-09-09` API version

File: specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2021-09-09/examples/ImportImageFromPublicRegistry.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+  "parameters": {
+    "api-version": "2021-06-01-preview",
```
Comment (jaysterp) [inline specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2021-09-09/examples/ImportImageFromPublicRegistry.json:3]: @AntonioVT please update this to use the 2021-09 API version

File: specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2021-09-09/examples/ImportImageByTag.json (hunk 1 above)
Comment (AntonioVT) [reply specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2021-09-09/examples/ImportImageByTag.json]: Fixed! I missed updating the api-version for the files I missed.

File: specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2021-09-09/examples/ImportImageFromPublicRegistry.json (hunk 2 above)
Comment (AntonioVT) [reply specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2021-09-09/examples/ImportImageFromPublicRegistry.json]: Done

**Comment by fengzhou-msft** [comment]
> @fengzhou-msft There are a couple of issues where I will need guidance from the swagger team. There are two failures in the [Azure OpenAPI pipeline](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1136704&view=logs&s=eb9754a7-3885-5b5b-bd91-16c95dd7881e):
> 
> 1. [SDK azure-sdk-for-net failed](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1136704&view=logs&j=2be18a23-047c-5c9f-61d7-60a33437d1f1&t=90227b5e-3523-5803-3e16-3788cf693a16): For this GA API, we are not including some features from the past preview APIs, such as Transfer. However, we have checked in [Transfer tests](https://github.com/Azure/azure-sdk-for-net/blob/f6afdee16a1958fa25f030a32b1b7ad104c0ac13/sdk/containerregistry/Microsoft.Azure.Management.ContainerRegistry/tests/Tests/ContainerRegistryTests.cs#L315) in the .NET SDK repo. The tests are currently failing because some of these definitions are not included in the current swagger.
> 
> What is your guidance on this? Could we get an exception for this test? Our .NET SDK is not automatically published.
> 
> 1. [SDK azure-sdk-for-go-track2](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1136704&view=logs&j=8ffa0edc-f7b1-5569-f96e-74c3aa46b2c3&t=594c76f5-4dbb-5af4-b03c-273ed3bf551d): I'm unfamiliar with this stage, as it was usually skipped in previous PRs. It seems that it can't find a the Go Module, but the [SDK azure-sdk-for-go](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1136704&view=logs&j=0303ead4-2e3b-557b-303e-fe8735d4d18f) step succeeded (previous to running this step). Any guidance on this?
> 
> cc @jaysterp

You can ignore the dotnet SDK test failure, it will not block this PR. But make sure there are only expected failures.

@ArcturusZhang for go SDK questions.

**Comment by jorgecotillo** [comment]
What is your base api? you need to follow our documentation: https://armwiki.azurewebsites.net/rp_onboarding/process/api_review.html basically these steps:

1. Create your new folder (RP namespace, and/or  api version)
2. Copy and paste your baseline swagger (from your previous api version)
3. Rename the files to match your needs
4. This step is super important, now commit & push this first iteration
5. Start making changes to your swagger (add the new definitions)

Please abandon this PR and create one following these steps, otherwise I'll have to review your entire swagger and will cause in delays or me reviewing existing items.

**Pull request closed without merging**